src/Field/Repeater.php

```diff
@@ -103,10 +103,14 @@ protected function fetchFields($fieldName, Builder $builder)
         foreach ($builder->get() as $meta) {
             $id = $this->retrieveIdFromFieldName($meta->meta_key, $fieldName);
             $name = $this->retrieveFieldName($meta->meta_key, $fieldName, $id);
-            
+
```

Comment:
Nah, if you're fine with it, I think I'll be able to live with some extraneous whitespace. Merge ahead :)